geowebcache/azureblob/src/main/java/org/geowebcache/azure/AzureBlobStoreInfo.java

```diff
@@ -123,12 +123,14 @@ public void setPrefix(String prefix) {
     }
 
     /** @return The maximum number of allowed open HTTP connections. */
-    public Integer getMaxConnections() {
-        return maxConnections == null ? DEFAULT_CONNECTIONS : maxConnections;
+    public String getMaxConnections() {
+        return maxConnections == null
+                ? Integer.valueOf(DEFAULT_CONNECTIONS).toString()
```

Comment:
``String.valueOf(DEFAULT_CONNECTIONS)`` would be more straightforward.

Reply:
Fixed.